lms/djangoapps/certificates/models.py

```diff
@@ -1000,6 +1000,13 @@ class CertificateTemplate(TimeStampedModel):
         help_text=_(u'On/Off switch.'),
         default=False,
     )
+    language = models.CharField(
+        max_length=2,
+        choices=settings.ALL_LANGUAGES,
+        blank=True,
+        null=True,
+        help_text=_(u'Language of template')
```

Comment:
Minor, but it might be helpful to add a little more to the help text explaining that language is optional and that the template will only be used for courses in that language if it's set.

Reply:
How about this : 
"Only courses in the selected language will be rendered using this template. Optional."